### DISCUSSION TITLE
Do anyone know any alternatives ?

### DISCUSSION THREAD

**Discussion by Rasad0** [TheLastBen/fast-stable-diffusion#2205] [General]
I have been using colab for much time. But for recent disconnecting issues I tried paperspace but it always show that they are full on capacity. does anyone have any better free solution? or I might not be able to use sd ever again

**Comment by GoofyAi** [comment]
https://colab.research.google.com/drive/1wEa-tS10h4LlDykd87TF5zzpXIIQoCmq
you will have to download the checkpoints and loras every time but it can be fixed if you just copy all the direct download link in notepad. its fast also.

**Reply by Rasad0** [reply]
how to do that?

**Reply by GoofyAi** [reply]
go down you will see a custom url section . just paste the direct model download link there. it also has batchlink downloader inside the ui so you can also download model after running the ui.

**Reply by HelpANewbie** [reply]
Anyone has used this link and is working fine ?
